webpack.mix.js

```diff
@@ -0,0 +1,54 @@
+let mix = require('laravel-mix');
+
+/*
+ |--------------------------------------------------------------------------
+ | Mix Asset Management
+ |--------------------------------------------------------------------------
+ |
+ | Mix provides a clean, fluent API for defining some Webpack build steps
+ | for your Laravel application. By default, we are compiling the Sass
+ | file for your application, as well as bundling up your JS files.
+ |
+ */
+const assets = 'assets';
+const dist = 'dist';
+const node = 'node_modules'
+
+mix.js(`${node}/bootstrap/dist/js/bootstrap.min.js`, `${dist}/scripts`)
+mix.copy(`${node}/bootstrap/dist/css/bootstrap.min.css`, `${dist}/styles`)
+
```

Comment:
will need to add `bootstrap/dist/fonts/glyphicons` (currently glyphicons not working)